### PR TITLE
feature: adds modifier to `updatingByIdArrayMerger` to manage position of new items

### PR DESCRIPTION
**Pull request by daviscabral**
This PR adds support to `unshift` items to the array. It might be more interesting to have `sort` maybe?

### CODE REVIEW

File: seamless-immutable-mergers.js [hunk 1]
```diff
@@ -50,7 +50,10 @@
       var matchingCurrentIndex = currentMap[other[j][identifier]];
 
       if (matchingCurrentIndex === undefined) {
-        resultList.push(other[j]);
+        var modifier = config.modifier;
+        if (modifier !== 'push' && modifier !== 'unshift')
+          modifier = 'push'
```
Comment (crudh) [inline seamless-immutable-mergers.js]: End the line with a semicolon to match the rest of the code.

**Comment by crudh** [comment]
@daviscabral Looks good! And besides the semicolon, could you add a few lines to the documentation also? No need for a full example, just some additions to the text at the end of the merger documentation.

**Comment by crudh** [comment]
@daviscabral and regarding sort I think it's better left out. That is a pretty specific use case in combination with this merger. I want to keep these as generic example mergers since it's so easy to roll your own based on these for specific use cases.

**Comment by daviscabral** [comment]
@crudh Great, thanks - just updated the docs and the code - let me know if you want any other change there.

**Comment by crudh** [comment]
@daviscabral thanks!

**Pull request merged**